rocket_privacy_notice/en-US.md

```diff
@@ -1,54 +1,54 @@
+## <span class="privacy-header-firefox-rocket">Firefox Lite</span> <span class="privacy-header-policy">Privacy Notice</span>
 
 May 11, 2018
 {: datetime="2018-5-11" }
 
 ## At Mozilla, we believe that privacy is fundamental to a healthy internet.
 
-That’s why we build Firefox Rocket, and all our products, to give you greater control over the information you share online and the information you share with us. We strive to collect only what we need to improve Rocket for everyone.
+That’s why we build Firefox Lite, and all our products, to give you greater control over the information you share online and the information you share with us. We strive to collect only what we need to improve Lite for everyone.
 
-In this Privacy Notice, we explain what data Rocket shares and point you to settings to share even less. We also adhere to the practices outlined in the Mozilla [privacy policy](https://www.mozilla.org/privacy/) for how we receive, handle and share information we collect from Rocket.
+In this Privacy Notice, we explain what data Lite shares and point you to settings to share even less. We also adhere to the practices outlined in the Mozilla [privacy policy](https://www.mozilla.org/privacy/) for how we receive, handle and share information we collect from Lite.
 
-## Firefox Rocket by default shares data to:
+## Firefox Lite by default shares data to:
 
 ### Improve performance and stability for users everywhere
 
-* __Interaction data__: Rocket sends data about your interactions with Rocket to us (number of webpages visited; and session length) and Rocket features offered by Mozilla or our partners (such as interaction with Rocket search features and search partner referrals).
+* __Interaction data__: Lite sends data about your interactions with Lite to us (number of webpages visited; and session length) and Lite features offered by Mozilla or our partners (such as interaction with Lite search features and search partner referrals).
 
-* __Technical data__: Rocket sends data about your Rocket version and language; device operating system and hardware configuration; memory, basic information about crashes and errors; outcome of automated processes like updates, and activation to us. When Rocket sends data to us, your IP address is temporarily collected as part of our server logs.
+* __Technical data__: Lite sends data about your Lite version and language; device operating system and hardware configuration; memory, basic information about crashes and errors; outcome of automated processes like updates, and activation to us. When Lite sends data to us, your IP address is temporarily collected as part of our server logs.
 
-Read detailed documentation for Rocket [here](https://github.com/mozilla-tw/Rocket/wiki/Telemetry) or turn off “Send Usage Data” in your Settings.
+Read detailed documentation for Lite [here](https://github.com/mozilla-tw/Lite/wiki/Telemetry) or turn off “Send Usage Data” in your Settings.
```

Comment:
Will this URL be changing? So far this new url is a 404.

Reply:
@pmac, this is something Taipei team needs to update. I am ahead of them on this. Also got a message from Mika saying both references to Firefox Rocket and Rocket will be changed to Firefox Lite. I will update the file and make changes to the other locales. I will suggest to change the url reference as .../Firefox-Lite/... If there is no objection, I will implement all them.

Reply:
Sounds good. What's the timeline. Should I go ahead and file an issue in bedrock for the change? Is there a bug?

Reply:
Here is the bug https://bugzilla.mozilla.org/show_bug.cgi?id=1498613. The plan is to complete the brand change by Oct 22. Production push is on Nov 1. I I have cc'd Wesley on both bugs (l10n side too) so he can comment on the timeline in case there are changes.

Reply:
@peiying2 - Stella has requested content/logo updates the Indonesian version of the WNP.  She as very clear that we could not make those updates until/on November 1st as the official launch Firefox Lite is on that day. I don't know if that requirement extends here, but wanted to check.

Reply:
@ejregithub https://docs.google.com/document/d/1Ls2x0Gzcu4w_TohDkOFJYp7VOYlB8nmrl7w2ms75ioc/edit?ts=5bc0cb0d - scroll down to legal and SUMO section.